cdap-data-fabric/src/main/java/co/cask/cdap/data2/metrics/HBaseDatasetMetricsReporter.java

```diff
@@ -94,25 +98,37 @@ protected final ScheduledExecutorService executor() {
     return executor;
   }
 
-  private void reportHBaseStats() throws IOException {
-    Map<String, HBaseTableUtil.TableStats> tableStats = hBaseTableUtil.getTableStats(hAdmin);
+  private void reportHBaseStats() throws IOException, DatasetManagementException {
+    Map<TableId, HBaseTableUtil.TableStats> tableStats = hBaseTableUtil.getTableStats(conf, hAdmin);
```

Comment:
we don't need to pass in a cConf, because the hBaseTableUtil should already have access to an instance of it.
In the setCConf method of HBaseTableUtil, save it as a field, and then this won't be necessary.
(the setCConf method gets called before the hBaseTableUtil is injected).